complex.c

```diff
@@ -1249,17 +1249,26 @@ nucomp_real_p(VALUE self)
 
 /*
  * call-seq:
- *    cmp <=> object  ->  0, 1, -1, or nil
+ *   complex <=> object -> -1, 0, 1, or nil
+ *
+ * Returns:
+ *
+ * - <tt>self.real <=> object.real</tt> if both of the following are true:
+ *
+ *   - +self+.imag == 0.
+ *   - +object+.imag == 0. # Always true if object is numeric but not complex.
```

Comment:
Would it make sense to format all of this as code?

```suggestion
 *   - <tt>self.imag == 0</tt>.
 *   - <tt>object.imag == 0</tt>. # Always true if object is numeric but not complex.
```